etc/ProdOfflineConfiguration.py

```diff
@@ -913,7 +855,7 @@
     addDataset(tier0Config, dataset,
                do_reco=False,
                write_reco=False, write_aod=False, write_miniaod=False, write_dqm=False,
-               disk_node=None,
+               diskNode="T2_CH_CERN",
```

Comment:
Hi @tvami  if you want raw data on disk you need to add the following variable to each pd of interest `raw_to_disk=True,`.
Also keep in mind that the function `addDataset` use `disk_node` instead of `diskNode`

Reply:
@jhonatanamado ok what happens if I use `raw_to_disk=True,` but I dont specify `disk_node`? Will that automatically go somewhere?

Reply:
Will go to our [default](https://github.com/dmwm/T0/blob/e1565d8e533867a666eb4381831d6ad4c99c61b3/etc/ProdOfflineConfiguration.py#L197) `disk_node` which is `T1_US_FNAL_Disk`

Reply:
Thanks! I modified the PR btw